dashboard/app/controllers/registrations_controller.rb

```diff
@@ -138,6 +138,24 @@ def destroy
     head :no_content
   end
 
+  def parent_email_params
+    params.
+      require(:user).
+      tap do |user|
+        user[:parent_email_preference_email] = user[:parent_email]
+        user[:parent_email_preference_opt_in_required] = '1'
+        user[:parent_email_preference_request_ip] = request.ip
+        user[:parent_email_preference_source] = EmailPreference::ACCOUNT_SIGN_UP
```

Comment:
Do we need a new SOURCE_TYPE for changes made on the edit account page?

Reply:
Oh good call!